addons/skin.estouchy/language/resource.language.en_gb/strings.po

```diff
@@ -78,22 +78,27 @@ msgctxt "#31016"
 msgid "Albums"
 msgstr ""
 
+#: /xml/DialogPlayerProcessInfo.xml
 msgctxt "#31017"
-msgid "PVR info"
+msgid "PVR"
 msgstr ""
 
+#: /xml/DialogPlayerProcessInfo.xml
 msgctxt "#31018"
-msgid "Player process info"
+msgid "Player"
```

Comment:
You got 2 labels of string "Player" now as you also added 31611

Reply:
oversight thanks fixed